### PR TITLE
general cleanup

### PR DESCRIPTION
**Pull request by anselmbradford**
## Changes

- General code style cleanup: consistent space around pipes, add TODO for todo comment, fix missing code comment info and formatting, eof returns, indentation issues.

## Testing

- `gulp build && gulp test:acceptance --sauce=false`

## Review

- @jimmynotjim 
- @sebworks 
- @KimberlyMunoz 
- @kave

### CODE REVIEW

File: cfgov/cfgov/settings/base.py [hunk 1]
```diff
@@ -27,7 +27,7 @@
     'wagtail.wagtailusers',
     'wagtail.wagtailimages',
     'wagtail.wagtailembeds',
-#    'wagtail.wagtailsearch', conflicts with haystack, will need to revist
+#    'wagtail.wagtailsearch', # TODO: conflicts with haystack, will need to revist.
```
Comment (KimberlyMunoz) [inline cfgov/cfgov/settings/base.py]: `revisit` instead of `revist` while you're in here.

**Comment by KimberlyMunoz** [comment]
:+1:

**Comment by jimmynotjim** [comment]
:+1: Pending Kim's note

File: cfgov/cfgov/settings/base.py (hunk 1 above)
Comment (anselmbradford) [inline cfgov/cfgov/settings/base.py]: Fixed!

**Pull request merged**